osu.Game.Tests/Visual/TestCaseNotificationOverlay.cs

```diff
@@ -91,19 +96,58 @@ private void sendProgress2()
 
         private void sendProgress1()
         {
-            var n = new ProgressNotification { Text = @"Uploading to BSS..." };
+            var n = new ProgressNotification(@"Uploading to BSS...", FontAwesome.fa_upload);
             manager.Post(n);
             progressingNotifications.Add(n);
         }
 
+        private void sendUpdateProgressNotification()
+        {
+            var completeNotification = new Notification("You are now running osu!lazer {version}.\nClick to see what's new!")
+            {
+                NotificationIcon = new NotificationIcon(FontAwesome.fa_check_square, osuColour.BlueDark)
+            };
+
+            var restartNotification = new Notification(@"Update ready to install. Click to restart!")
+            {
+                NotificationIcon = new NotificationIcon(backgroundColour: osuColour.Green)
+            };
+            restartNotification.OnActivate += () => manager.Post(completeNotification);
+
+            bool downloadCompleted = false;
+            var downloadNotification = new ProgressNotification(@"Downloading update...")
+            {
+                NotificationIcon = new NotificationIcon(
+                    FontAwesome.fa_upload,
+                    ColourInfo.GradientVertical(osuColour.YellowDark, osuColour.Yellow)
+                )

```

Comment:
Inline the construction of this icon.

Reply:
https://i.imgur.com/eR6X2kh.png this feels to long for a line